frontend/packages/console-shared/src/components/actions/menu/ActionMenuContent.tsx

```diff
@@ -151,12 +151,14 @@ const ActionMenuContent: React.FC<ActionMenuContentProps> = ({ options, onClick,
               );
             default:
               return (
-                <ActionMenuItem
-                  key={option.id}
-                  action={option as Action}
-                  onClick={onClick}
-                  autoFocus={focusItem ? option === focusItem : undefined}
-                />
+                <div className="pf-c-dropdown__menu-item">
```

Comment:
Added this so that the menu items in the context menu are rendered with proper spacing and are focused when the cursor hovers on them

Reply:
This might add some unnecessary styles to normal menu items.